src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,13 +1,18 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Truck
- * Do not remove no-field constructor
- */
 public class Truck extends Machine {
+    private int size;
+    private String typeOfEngine;
+
     public Truck() {
     }
```

Comment:
If you dont use this constructor, you can remove it

Reply:
I can't delete this constructor because it needs so that run tests